test/Microsoft.ApplicationInsights.AspNet.Tests/FakeTelemetryChannel.cs

```diff
@@ -6,6 +6,8 @@
     // TODO: Remove FakeTelemetryChannel when we can use a dynamic test isolation framework, like NSubstitute or Moq
```

Comment:
I just tried NSubstitute, which works with dnx451. I'm pretty sure it's because it supports framework net45.Moq will be the same. We can start using either one if we remove the dnxcore50 framework from our test projects. Do we want to do that?